tests/test_keep_alive_timeout.py

```diff
@@ -9,14 +9,39 @@
 from aiohttp import TCPConnector
 from sanic.testing import SanicTestClient, HOST, PORT
 
+try:
+    try:
+        import packaging # direct use
+    except ImportError:
+        # setuptools v39.0 and above.
+        try:
+            from setuptools.extern import packaging
+        except ImportError:
+            # Before setuptools v39.0
+            from pkg_resources.extern import packaging
+    version = packaging.version
+except ImportError:
+    raise RuntimeError("The 'packaging' library is missing.")
```

Comment:
haha this is rough, dumb style question, does this seem more approchable?
```

try:
    import packaging # direct use
except ImportError:
    import setuptools
    major = int(setuptools.__version__.partition('.')[0])
    # support setup tools pre and post version 39
    if major > 39:
        from setuptools.extern import packaging
    else:
        from pkg_resources.extern import packaging
version = packaging.version
```
flattens the code just a bit and expresses the comments as code but I think your is equally valid